### ISSUE TITLE
Support annotations with members

### ISSUE THREAD

**Issue by johnlinp** [plantuml/plantuml#458]
I want to write an annotation with members in PlantUML. Specifically, I want to describe this Java code in PlantUML:

```java
public @interface Foo {
    String bar();
    String baz();
}
```

I tried to write it in PlantUML, but it causes syntax error:

```
@startuml
annotation Foo {
    String bar()
    String baz()
}
@enduml
```

http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuKhCoyilIIp9pCzJSClFLwZcKW22u9AYpBnqXQJ48WrDL84ge40jbqDgNWfGCm00

Can PlantUML support annotations with members? Thanks!

**Comment by arnaudroques** [comment]
Thanks for the feedback.
This is now possible with last beta http://beta.plantuml.net/plantuml.jar
Is this what you were looking for ?

**Comment by johnlinp** [comment]
Yes, exactly! Thank you for letting me know. May I ask when this feature will be publicly available?

**Comment by arnaudroques** [comment]
This has just been released in V1.2020.2

**Comment by johnlinp** [comment]
I tried it on the online server without problems. Thanks you so much.
